### ISSUE TITLE
Certificate issue in grafana-image-renderer

### ISSUE THREAD

**Issue by kozsoo33** [grafana/grafana#25140]
Hi,
I have just upgrade to 7.0 and lost the image rendering.
I tried install render plugin with grafana-cli, no success, maybe has no fw rules for it.

<pre>
[root@tool chrome-linux]#  grafana-cli plugins install grafana-image-renderer
Error: â Failed to send request: Get "https://grafana.com/api/plugins/repo/grafana-image-renderer": proxyconnect tcp: dial tcp: lookup https on 172.28.18.1:53: no such host
</pre>

I made manual install, downloading the plugin.
I can see the plugin as unsigned and not sure if it is loaded properly.

<pre>
t=2020-05-27T10:27:28+0200 lvl=info msg=Rendering logger=rendering renderer=plugin path="d-solo/GWrmds_iz/cdrcheck?orgId=1&from=1587971691483&to=1590563691484&panelId=2&width=1000&height=500&tz=Europe%2FBudapest"
t=2020-05-27T10:27:28+0200 lvl=dbug msg="Calling renderer plugin" logger=rendering renderer=plugin req="url:\"https://localhost:8443/d-solo/GWrmds_iz/cdrcheck?orgId=1&from=1587971691483&to=1590563691484&panelId=2&width=1000&height=500&tz=Europe%2FBudapest&render=1\" width:1000 height:500 deviceScaleFactor:1 filePath:\"/var/lib/grafana/png/lyCmk1WKeCBsT5YgJppl.png\" renderKey:\"nFC10cKzKAq2t34pgzOWWqfeP1DA9iCY\" domain:\"localhost\" timeout:60 timezone:\"Europe/Budapest\" headers:<key:\"Accept-Language\" value:<values:\"hu-HU,hu;q=0.9,en-US;q=0.8,en;q=0.7\" > > "
t=2020-05-27T10:27:28+0200 lvl=dbug msg="Render request received" logger=plugins.backend pluginId=grafana-image-renderer url="https://localhost:8443/d-solo/GWrmds_iz/cdrcheck?orgId=1&from=1587971691483&to=1590563691484&panelId=2&width=1000&height=500&tz=Europe%2FBudapest&render=1"
t=2020-05-27T10:27:29+0200 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=5
t=2020-05-27T10:27:29+0200 lvl=dbug msg="Setting extra HTTP headers for page" logger=plugins.backend pluginId=grafana-image-renderer headers="map[Accept-Language:hu-HU,hu;q=0.9,en-US;q=0.8,en;q=0.7]"
t=2020-05-27T10:27:29+0200 lvl=eror msg="Browser request failed" logger=plugins.backend pluginId=grafana-image-renderer url="https://localhost:8443/d-solo/GWrmds_iz/cdrcheck?orgId=1&from=1587971691483&to=1590563691484&panelId=2&width=1000&height=500&tz=Europe%2FBudapest&render=1" method=GET failure=net::ERR_CERT_AUTHORITY_INVALID
t=2020-05-27T10:27:29+0200 lvl=eror msg="Render request failed" logger=plugins.backend pluginId=grafana-image-renderer url="https://localhost:8443/d-solo/GWrmds_iz/cdrcheck?orgId=1&from=1587971691483&to=1590563691484&panelId=2&width=1000&height=500&tz=Europe%2FBudapest&render=1" error="Error: net::ERR_CERT_AUTHORITY_INVALID at https://localhost:8443/d-solo/GWrmds_iz/cdrcheck?orgId=1&from=1587971691483&to=1590563691484&panelId=2&width=1000&height=500&tz=Europe%2FBudapest&render=1"
t=2020-05-27T10:27:29+0200 lvl=eror msg="Rendering failed." logger=context userId=1 orgId=1 uname=admin error="Rendering failed: Error: net::ERR_CERT_AUTHORITY_INVALID at https://localhost:8443/d-solo/GWrmds_iz/cdrcheck?orgId=1&from=1587971691483&to=1590563691484&panelId=2&width=1000&height=500&tz=Europe%2FBudapest&render=1"
t=2020-05-27T10:27:29+0200 lvl=dbug msg="token needs rotation" logger=auth tokenId=220 authTokenSeen=true rotatedAt=2020-05-27T09:10:54+0200
t=2020-05-27T10:27:29+0200 lvl=dbug msg="auth token rotated" logger=auth affected=1 auth_token_id=220 userId=1
t=2020-05-27T10:27:29+0200 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=GET path=/render/d-solo/GWrmds_iz/cdrcheck status=500 remote_addr=10.201.141.8 time_ms=1737 size=1723 referer="https://tool.xxxxxx.intra:8443/d/GWrmds_iz/cdrcheck?orgId=1"
</pre>

I have tried to add internal or root CA to the nssdb, no success.
(I have signed cert by company, company internal CA, company root CA) 

Tried with these configs also:
<pre>
[plugins]
allow_loading_unsigned_plugins = grafana-image-renderer

[log]
filters = rendering:debug

[rendering]
callback_url = https://tool.xxxxxx.intra
rendering_ignore_https_errors = true
rendering_verbose_logging = debug
</pre>

Should ignore_https_errors ignore the certificate problem also?
I can see "ignoresHttpsErrors:false" instead of true.

<pre>
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing PluginManager" logger=server
t=2020-05-27T10:51:28+0200 lvl=info msg="Starting plugin search" logger=plugins
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Scanning core plugin directory" logger=plugins dir=/usr/share/grafana/public/app/plugins
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Scanning bundled plugins directory" logger=plugins dir=/usr/share/grafana/plugins-bundled
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Transform plugin is disabled since the expressions feature toggle is not enabled" logger=plugins pluginID=gel
t=2020-05-27T10:51:28+0200 lvl=info msg="Registering plugin" logger=plugins name="Direct Input"
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Scanning external plugins directory" logger=plugins dir=/var/lib/grafana/plugins
t=2020-05-27T10:51:28+0200 lvl=info msg="Registering plugin" logger=plugins name="Graph Compare Panel"
t=2020-05-27T10:51:28+0200 lvl=info msg="Registering plugin" logger=plugins name="Grafana Image Renderer"
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Registering backend plugin" logger=plugins.backend pluginId=grafana-image-renderer executablePath=/var/lib/grafana/plugins/plugin-linux-x64-glibc/plugin_start_linux_amd64
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Backend plugin registered" logger=plugins.backend pluginId=grafana-image-renderer executablePath=/var/lib/grafana/plugins/plugin-linux-x64-glibc/plugin_start_linux_amd64
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Getting signature state of plugin" logger=plugins plugin=input
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Verifying contents of plugin manifest" logger=plugins plugin=input
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Getting signature state of plugin" logger=plugins plugin=grafana-image-renderer
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Getting signature state of plugin" logger=plugins plugin=graph-compare-panel
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing AlertEngine" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing QuotaService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing ServerLockService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing UserAuthTokenService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing DatasourceCacheService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing LoginService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing provisioningServiceImpl" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Looking for alert notification provisioning files" logger=provisioning.notifiers path=/etc/grafana/provisioning/notifiers
t=2020-05-27T10:51:28+0200 lvl=eror msg="Can't read alert notification provisioning files from directory" logger=provisioning.notifiers path=/etc/grafana/provisioning/notifiers error="open /etc/grafana/provisioning/notifiers: no such file or directory"
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing InternalMetricsService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing TracingService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing UsageStatsService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing CleanUpService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing NotificationService" logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Looking for App Dashboard Updates" logger=plugins
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Plugins: Adding route" logger=http.server route=/public/plugins/graph-compare-panel dir=/var/lib/grafana/plugins/graph-compare-panel-master/dist
t=2020-05-27T10:51:28+0200 lvl=dbug msg="starting plugin" logger=plugins.backend pluginId=grafana-image-renderer path=/var/lib/grafana/plugins/plugin-linux-x64-glibc/plugin_start_linux_amd64 args=[/var/lib/grafana/plugins/plugin-linux-x64-glibc/plugin_start_linux_amd64]
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Plugins: Adding route" logger=http.server route=/public/plugins/input dir=/usr/share/grafana/plugins-bundled/internal/input-datasource
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Plugins: Adding route" logger=http.server route=/public/plugins/grafana-image-renderer dir=/var/lib/grafana/plugins/plugin-linux-x64-glibc
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Checking for updates" logger=plugins
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Initializing Stream Manager"
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Found old rendered image to delete" logger=cleanup deleted=0 kept=0
t=2020-05-27T10:51:28+0200 lvl=info msg="HTTP Server Listen" logger=http.server address=[::]:8443 protocol=https subUrl= socket=
t=2020-05-27T10:51:28+0200 lvl=dbug msg="Waiting on services..." logger=server
t=2020-05-27T10:51:28+0200 lvl=dbug msg="plugin started" logger=plugins.backend pluginId=grafana-image-renderer path=/var/lib/grafana/plugins/plugin-linux-x64-glibc/plugin_start_linux_amd64 pid=5493
t=2020-05-27T10:51:28+0200 lvl=dbug msg="waiting for RPC address" logger=plugins.backend pluginId=grafana-image-renderer path=/var/lib/grafana/plugins/plugin-linux-x64-glibc/plugin_start_linux_amd64
t=2020-05-27T10:51:29+0200 lvl=dbug msg="Browser initialized" logger=plugins.backend pluginId=grafana-image-renderer config="map[args:[--no-sandbox] chromeBin:/var/lib/grafana/plugins/plugin-linux-x64-glibc/chrome-linux/chrome clustering:map[maxConcurrency:5 mode:browser] deviceScaleFactor:1 dumpio:false height:500 ignoresHttpsErrors:false maxDeviceScaleFactor:3 maxHeight:3000 maxWidth:3000 mode:default verboseLogging:false width:1000]"
t=2020-05-27T10:51:29+0200 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=5
t=2020-05-27T10:51:29+0200 lvl=dbug msg="Using browser version" logger=plugins.backend pluginId=grafana-image-renderer browserVersion=HeadlessChrome/79.0.3945.0
t=2020-05-27T10:51:29+0200 lvl=dbug msg="using plugin" logger=plugins.backend pluginId=grafana-image-renderer version=2

</pre>

Any idea or detailed guide is needed, thx.

_Originally posted by @kozsoo33 in https://github.com/grafana/grafana/issues/19842#issuecomment-634527420_

**Comment by marefr** [comment]
Closing this in favor of comment in grafana-image-renderer repo.
